Description: Add zmq.ssh and zmq.utils.z85 modules to zmq namespace
   For some reason those don't exist as is. Import them in __init__.py.
Author: Goswin von Brederlow <brederlo@q-leap.de>
Last-Update: 2014-01-14

---

Index: pyzmq/zmq/__init__.py
===================================================================
--- pyzmq.orig/zmq/__init__.py	2014-05-20 11:24:17.252165753 +0200
+++ pyzmq/zmq/__init__.py	2014-05-20 11:24:52.150601396 +0200
@@ -61,6 +61,8 @@
 from zmq import sugar
 from zmq.sugar import *
 from zmq import devices
+import zmq.utils.z85 as z85
+from zmq import ssh
 
 def get_includes():
     """Return a list of directories to include for linking against pyzmq with cython."""
